webpack/common/rtlTestHelpers.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
the original `rtlHelpers` from [core](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/rtlTestHelpers.js) should be available to the plugins. 
@MariaAga You propbably know the best way to reference it from the plugin